src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -59,6 +60,11 @@ internal sealed class InProcRemoteHostClient : RemoteHostClient
 
         public AssetStorage AssetStorage => _inprocServices.AssetStorage;
 
+        public void RegisterService(string name, Func<Stream, IServiceProvider, ServiceHubServiceBase> serviceCreator)
```

Comment:
that is my test mock. I changed the mock to let me to register test service so that I can call it through remote host client like CodeAnalysisService and etc.